File: Java/src/com/leetcode/Easy/ThreeConsecutiveOdds.java
package com.leetcode.Easy;

// 1550. Three Consecutive Odds
public class ThreeConsecutiveOdds {
    public boolean threeConsecutiveOdds(int[] arr) {
        int count = 0;
        for (int i : arr) {
            if (i % 2 != 0)
                count++;
            else
                count = 0;

            if (count == 3)
                return true;
        }
        return false;
    }
}

File: JavaScript/Easy/ThreeConsecutiveOdds.js
/**
 * 1550. Three Consecutive Odds
 * @param {number[]} arr
 * @return {boolean}
 */
const threeConsecutiveOdds = function (arr) {
  let countOddNumbers = 0;
  for (let i = 0; i < arr.length; i++) {
    if (arr[i] % 2 != 0) {
      countOddNumbers++;
    } else {
      countOddNumbers = 0;
    }

    if (countOddNumbers === 3)
      return true;
  }

  return false;
};